### ISSUE TITLE
Generate file per component

### ISSUE THREAD

**Issue by nosuchip** [zouhir/preact-habitat#34]
If application exposes dozen ob components it would be nice to build one bundle per component rather then global huge bundle with all components. Users may want to embed single block, not all.

**Comment by gijo-varghese** [comment]
@nosuchip did you find any solution? I'm looking for the same

**Comment by nosuchip** [comment]
@gijo-varghese nope, I tried many different approaches and finally decided that it is not possible at least without radical changes in webpack. I am not ready to dive into this stuff so deep.

**Comment by gijo-varghese** [comment]
@nosuchip hmmm. ok

**Comment by zouhir** [comment]
I am looking at migrating the entire project & example to Vite. I don't know what I don't know yet about Vita configurations but I will keep this request in mind during my investigations.

**Comment by simonv3** [comment]
@zouhir did you get this migrated to vite? Or any insights? We're using preact-habitat with a project we just migrated to vite and are looking at now accomodating preact-habitat's insertion in the build process
